doc/release-notes-0.4.0.rst

```diff
@@ -163,7 +163,7 @@ Profiler
 Build and configuration system
 ==============================
 
-Bootloaders (such as SPM) and MCUboot no longer need to be built and flashed separately, but are instead automatically built and flashed (if enabled by the application).
+Bootloaders (such as SPM) and MCUboot no longer need to be built and programmed separately, but are instead automatically built and programmed (if enabled by the application).
```

Comment:
These are old release notes - I think we should not modify them (unless we need to because of, for example, breaking links). Let's leave that file as it is. :)

Reply:
Reverted the changes.